pkg/alertmanager/alertmanager_metrics.go

```diff
@@ -209,6 +212,18 @@ func newAlertmanagerMetrics() *alertmanagerMetrics {
 			"cortex_alertmanager_notification_rate_limited_total",
 			"Total number of rate-limited notifications per integration.",
 			[]string{"user", "integration"}, nil),
+		insertAlertFailures: prometheus.NewDesc(
+			"cortex_alertmanager_alerts_insert_failures_total",
```

Comment:
Should we call it `cortex_alertmanager_alerts_insert_rate_limited_total` similarly to `alertmanager_notification_rate_limited_total`?

Reply:
`cortex_alertmanager_alerts_insert_rate_limited_total` is not strictly true... there is no rate limit here, just a limit for number of alerts or their size. What about `cortex_alertmanager_alerts_insert_limited_total` instead? (dropping "rate" part)

Reply:
> What about cortex_alertmanager_alerts_insert_limited_total instead?

LGTM